Makefile

```diff
@@ -67,14 +126,17 @@ $(DIST)/libnetwork-plugin: vendor
 		-e ARCH=$(ARCH) \
 		$(GO_BUILD_CONTAINER) sh -c '\
 			cd /go/src/github.com/projectcalico/libnetwork-plugin && \
-			make build'
+			make binary'
 
-build: $(SRC_FILES) vendor
-	CGO_ENABLED=0 go build -v -i -o $(DIST)/libnetwork-plugin -ldflags "-X main.VERSION=$(shell git describe --tags --dirty) -s -w" main.go
+binary: $(SRC_FILES) vendor
+	CGO_ENABLED=0 GOARCH=$(ARCH) go build -v -i -o $(DIST)/libnetwork-plugin -ldflags "-X main.VERSION=$(shell git describe --tags --dirty) -s -w" main.go
```

Comment:
Release default: If you look at the [releases page](https://github.com/projectcalico/libnetwork-plugin/releases), you see that the unnamed `libnetwork-plugin` is assumed to be `amd64`. I think we need to have in the `dist/` dir, in addition to `libnetwork-plugin-<arch>`, a single file `libnetwork-plugin` which is `amd64`. As @caseydavenport pointed out, we eventually should change our documentation and other processes to eliminate that, but it is _far_ easier to add all of the archs as part of the release and a default "archless" file which is just `cp libnetwork-plugin-amd64 libnetwork-plugin`, and then later change the dependent docs and installs. 

The above just needs something like:

```make
ifeq ($(ARCH),amd64)
     cp dist/libnetwork-plugin-amd64 dist/libnetwork-plugin
endif
```